.github/workflows/build.yml

```
@@ -382,7 +458,22 @@ jobs:
             --build \
             --parallel \
             --build_shared_lib \
-            "${build_opts[@]}"
+            "${build_opts[@]}" \
+            1>${{ env.ONNXRUNTIME_NAME == 'onnxruntime' && '&1' || format('{0}_stdout.txt', matrix.artifact_name) }} \
+            2>${{ env.ONNXRUNTIME_NAME == 'onnxruntime' && '&2' || format('{0}_stderr.txt', matrix.artifact_name) }}
+
+      - name: Send build log to Discord
+        if: failure() && steps.build.outcome == 'failure' && env.ONNXRUNTIME_NAME == 'voicevox_onnxruntime'
+        run: |
+          curl \
+            "$DISCORD_WEBHOOK" \
+            -sf \
+            -F 'payload_json={"username":"onnxruntime-builder"}' \
+            -F file1=@${{ matrix.artifact_name }}_stdout.txt \
+            -F file2=@${{ matrix.artifact_name }}_stderr.txt \
+            > /dev/null

```

過剰に臆病かもですが、stderrも/dev/null送りにした方がいいかも。
```suggestion
            > /dev/null 2>&1

```

---

fixup: [`3a40379` (#52)](https://github.com/VOICEVOX/onnxruntime-builder/pull/52/commits/3a40379820eaadab07a083d7da31ef97f352287b)